servers/src/proto/journal/file.proto

```diff
@@ -101,10 +101,19 @@ message RenameEntry {
 }
 
 // next available id: 6
+message SetAclEntry {
+  optional int64 id = 1;
+  optional int64 op_time_ms = 2;
+  optional string user_name = 3;
+  optional string group_name = 4;
+  optional int32 permission = 5;
+}
+
+// next available id: 6
 message SetStateEntry {
   optional int64 id = 1;
   optional int64 op_time_ms = 2;
   optional bool pinned = 3;
   optional int64 ttl = 4;
   optional bool persisted = 5;
-}
```

Comment:
we usually keep the EOF on a separate line (revert back to the original line 110)